src/main/java/data/QueryInformation.java

```diff
@@ -27,29 +27,29 @@
   private static final long MILLISECONDS_TO_END_OF_DAY = (24L * 60L * 60L * 1000L) - 1L;
 
   private final TimeZone timeZone;
-  private final ImmutableSet<String> categories;
+  private final ImmutableSet<String> category;
```

Comment:
nit: if this is a list we should keep plural `categories`

Reply:
I had it as categories, but in #32 I was told to use category since for the mvp only one category is accepted. Should I still change it back? I could also change it back to categories in the next iteration